layouts/shortcodes/cve-feed.html

```diff
@@ -21,3 +22,43 @@
   {{ end }}
   </tbody>
 </table>
+
+<script>
+  window.addEventListener("DOMContentLoaded", () => {
```

Comment:
💭 How about adding a refresh/fetch button in the corner of the table, alongside this auto-trigger logic of every 10 minutes? This button could be used explicitly to refresh the feed table data.